### DISCUSSION TITLE
GHSA & NVD redundancy

### DISCUSSION THREAD

**Discussion by andreeaButerchi** [DependencyTrack/dependency-track#4013] [Q&A]
Hello,

As NVD analysis takes longer and longer we activated the GHSA as Vulnerability Source (besides NVD)... But we end up having several times the same vulnerability:

- once from the GHSA 
- and once more from NVD
Taking into account that if we check the GHSA ID we can find the reference of the NVD CVE. We were expecting to be able to get only once a unique vulnerability.
For example:

![image](https://github.com/user-attachments/assets/fa4215b5-f0d0-4161-83a6-b996f12d5a95)

we have: https://github.com/advisories/GHSA-hr8g-6v94-x4m9
![image](https://github.com/user-attachments/assets/fe0c2fae-b872-4ead-9c1e-0d57cb376556)

And also: https://github.com/advisories/GHSA-8xfc-gm6g-vgpv
![image](https://github.com/user-attachments/assets/0bf02304-2024-4136-ab58-d8814bdab81d)

I hope I'm missing some sort of configuration... or API option (currently we're using variant=withVulnerabilities)

Thank you very much for your help!

Best regards,
Andreea

**Comment by andreeaButerchi** [comment]
Hello! Any chance someone ran into a similar situation? and what would the recommended solution be?

**Reply by peter-nicholson** [reply]
Did you ever get an answer on this? I am seeing the same behaviour.
